docs/xrsonrestoresynconrestore.md

```diff
@@ -0,0 +1,55 @@
+﻿<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -Xrs:[onRestore|syncOnRestore]
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+The `-Xrs` option is an existing option and `onRestore` and `syncOnRestore` are its new parameters that can be used for controlling the signal handling at the time of restore if the CRIU feature is available.
+
+If the VM was started with signal handling enabled, then at the time of restore, the signal handling is enabled by default. You can use these options to disable signal handling on restore. These options are required if there are already some active signal handlers including the global signal handlers existing on restore and you want to prevent the addition of new signal handlers.
```

Comment:
Usual comment about passive voice, eg "when you restore the VM" rather than "at the time of restore". If you make the earlier change, you don't need the sentence starting "You can...". The last sentence is unclear and needs improvement or just removing altogether and replacing with a sentence that explains that these suboptions are equivalent to existing ones but with differences, as explained in the doc issue.